### ISSUE TITLE
How to specify SQLite database fullpath as database URL

### ISSUE THREAD

**Issue by otsuka** [encode/databases#115]
When I specify an absolute path of SQLite database file as database url as below, aiosqlite cannot open the database file. (on MacOS)
When I use a relative path like `sqlite:///example.db`, it works fine.
How could I specify the absolute path?

```
database = databases.Database('sqlite:////Users/otsuka/path/to/example.db')
await database.connect()

await database.fetch_all('select * from users')

.venv/lib/python3.7/site-packages/aiosqlite/core.py in connector()
    300             loc = str(database)
    301 
--> 302         return sqlite3.connect(loc, **kwargs)
    303 
    304     return Connection(connector, loop)

OperationalError: unable to open database file
```

**Comment by tomchristie** [comment]
Looks like we've got an open pull request for this in #95.

The pull request is currently failing due to missing test coverage. In order to get this resolved someone need to pick up the pull request and add in the missing coverage to bring it to 100%.

**Comment by mivade** [comment]
This is a pretty serious limitation. Can we get an update on this? It seems like a pretty trivial update.

**Comment by tomchristie** [comment]
Available now, in version 0.2.4

**Comment by mivade** [comment]
Thanks!
